pyOCD/transport/cmsis_dap_core.py

```diff
@@ -74,348 +74,350 @@
 
 MAX_PACKET_SIZE = 0x0E
 
-def dapInfo(interface, id_):
-    cmd = []
-    cmd.append(COMMAND_ID['DAP_INFO'])
-    cmd.append(ID_INFO[id_])
-    interface.write(cmd)
-
-    resp = interface.read()
-    if resp[0] != COMMAND_ID['DAP_INFO']:
-        raise ValueError('DAP_INFO response error')
-    
-    if resp[1] == 0:
+## @brief This class implements the CMSIS-DAP wire protocol.
+class CMSIS_DAP_Protocol(object):
```

Comment:
Was this made into a class in preparation for another protocol similar to CMSIS_DAP?  If so you might want to add an interface for it.